pkg/preparer/orchestrate.go

```diff
@@ -203,6 +219,11 @@ func (p *Preparer) verifySignature(manifest pods.PodManifest, logger logging.Log
 func (p *Preparer) installAndLaunchPod(newManifest *pods.PodManifest, pod Pod, logger logging.Logger) bool {
 	// do not remove the logger argument, it's not the same as p.Logger
 
+	if p.podIdForbidden(newManifest.ID()) {
```

Comment:
My first nomination for the thing that needs to be moved into a separate function, actually... 
RunHookType, or warn if unable